salesforce/client.bal

```diff
@@ -69,6 +65,250 @@ public isolated client class Client {
         }
     }
 
+    //Describe SObjects
+    # Get metadata for your organization.
+    #
+    # + return - `OrganizationMetadata` record if successful or else `sfdc:Error`
```

Comment:
This should be error. Check all places

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sfdc/commit/666dfedf90963e702c3e32712fa8e2e41f087f2e